packages/slate/src/utils/memoize.js

```diff
@@ -79,12 +77,19 @@ function memoize(object, properties, options = {}) {
         if (CACHE_KEY !== this.__cache_key) {
           this.__cache_key = CACHE_KEY
           this.__cache = new Map() // eslint-disable-line no-undef,no-restricted-globals
+          this.__cache_no_args = {}
         }
       }
 
       if (!this.__cache) {
         this.__cache = new Map() // eslint-disable-line no-undef,no-restricted-globals
       }
+      if (!this.__cache_no_args) {
+        this.__cache_no_args = {}
+      }
+
+      // The default parameter of text.getLeaves would have an arg with fn.length===0
```

Comment:
Can you remove this comment that is no longer relevant?

Reply:
Fixed~ Thank you.